### PR TITLE
Slider remove commons

### PR DESCRIPTION
**Pull request by micahgodbolt**
as per #22862

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/b8afea24b4adce2fcc534204dc06fc0467309f25'>b8afea24b4adce2fcc534204dc06fc0467309f25</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22879/perf-test-react-components/Avatar-mount_base.html">1290</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22879/perf-test-react-components/Avatar-mount.html">1324</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22879/perf-test-react-components/Button-mount_base.html">788</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22879/perf-test-react-components/Button-mount.html">832</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22879/perf-test-react-components/FluentProvider-mount_base.html">2357</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22879/perf-test-react-components/FluentProvider-mount.html">2366</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22879/perf-test-react-components/FluentProviderWithTheme-mount_base.html">388</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22879/perf-test-react-components/FluentProviderWithTheme-mount.html">396</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22879/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">363</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22879/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">366</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22879/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">453</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22879/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">452</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22879/perf-test-react-components/MakeStyles-mount_base.html">2046</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22879/perf-test-react-components/MakeStyles-mount.html">2017</a></td>
            <td>50000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

File: packages/react-components/react-slider/src/components/Slider/Slider.types.ts
Comment (spmonahan) [inline packages/react-components/react-slider/src/components/Slider/Slider.types.ts:46]: @smhigley has been alphabetizing props as part of this change. Can you update these to be alphabetized?

File: packages/react-components/react-slider/src/components/Slider/Slider.types.ts
Comment (smhigley) [inline packages/react-components/react-slider/src/components/Slider/Slider.types.ts:113]: same alphabetical ordering thing here.

But also -- do we need all these on state? Looking at Slider, it seems that the only values actually used on `state` (aside from slots) are `disabled`, `size`, and `vertical`.

File: packages/react-components/react-slider/src/components/Slider/Slider.types.ts
Comment (smhigley) [inline packages/react-components/react-slider/src/components/Slider/Slider.types.ts:86]: origin is still out of order, I think 🙈

**Review by smhigley — APPROVED** [review]
there's a single prop still out of alphabetical order, but I'm approving now so you can merge right after updating it. (And it's also not that important :D)

File: packages/react-components/react-slider/src/components/Slider/Slider.types.ts
Comment (micahgodbolt) [reply packages/react-components/react-slider/src/components/Slider/Slider.types.ts:86]: i can't do alphabet good

**Review by spmonahan — APPROVED** [review]

**Pull request merged**
